cmd/crank/configuration/configuration.go

```diff
@@ -96,12 +103,35 @@ type CreateCmd struct {
 	Name    string `arg:"" name:"name" help:"Name of Configuration."`
 	Package string `arg:"" name:"package" help:"Image containing Configuration package."`
 
-	History  int  `help:"Revision history limit."`
-	Activate bool `short:"a" help:"Enable automatic revision activation policy."`
+	RevisionHistoryLimit int64 `help:"Revision history limit."`
```

Comment:
Should we maybe add a short flag here given that changing the name from `History` makes the verbose flag much longer?

Reply:
How does `rl` sound?

Reply:
SGTM